### PR TITLE
[13.0][ADD] purchase_isolated_rfq

### PR DESCRIPTION
**Pull request by kongrattapong**
This module base on Module : sale_isolated_quotation

**Description:**
In some countries/companies, It's already common to separate these two documents.
For filing purposes, the document sequence of Requests For Quotation (RFQ) and Purchases order
has to be separated. In practice, there could be multiple RFQ open
to a vendor, yet only one RFQ get converted to the Purchases order.

This module separate RFQ and Purchases order by adding order_sequence flag in
purchase.order model.

Each type of document will have separated sequence numbering.
RFQ will have only 2 state, Draft and Done. Purchases Order work as normal.

cc. @kittiu @Oakyman

### CODE REVIEW

File: purchase_isolated_rfq/hooks.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+def post_init_hook(cr, registry):
+    """ Set value for order_sequence on old records """
+    cr.execute(
+        """
+        update purchase_order
+        set order_sequence = true
+        where state not in ('draft', 'cancel')
+    """
```
Comment (kittiu) [inline purchase_isolated_rfq/hooks.py]:
```suggestion
        """
```

Comment (kittiu) [reply purchase_isolated_rfq/hooks.py]: Just make it even.

File: purchase_isolated_rfq/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Base on sale_isolated_quotation
```
Comment (kittiu) [inline purchase_isolated_rfq/readme/DESCRIPTION.rst]:
```suggestion
Note: This module is similar to sale_isolated_quotation, but for purchase/rfq
```

**Review by kittiu — APPROVED** [review]
LGTM (excel some very small comments about wording)

**Comment by kongrattapong** [comment]
Thank you for your review. Fixed.

**Review by HviorForgeFlow — APPROVED** [review]
Code and Functional review
LGTM :+1:

**Comment by HviorForgeFlow** [comment]
@kittiu, is it ready for merge?

**Comment by kittiu** [comment]
@HviorForgeFlow yes, it is ready :)

**Pull request merged**